databricks/koalas/indexes.py

```diff
@@ -1218,11 +1218,9 @@ def argmax(self):
         >>> kidx.argmax()
         4
         """
-        sdf = self._internal.sdf
-        sdf = sdf.select(self._scol.alias('__index_value__'))
-
-        sdf, index_column = _InternalFrame.attach_default_index(
-            sdf, default_index_type='distributed-sequence')
+        sdf = self._internal.sdf.select(self._scol)
+        sequence_col = "__distributed_sequence_column__"
+        sdf = _InternalFrame.attach_distributed_sequence_column(sdf, column_name=sequence_col)
```

Comment:
ditto.

Reply:
Here's actually it adds a column that has a sequence with a temp column. Maybe it's better to have a method that generates a name automatically but that would make the debug harder with an arbitrary name.

Using index column `__index_level_0__` as a temp column wouldn't be a good idea. If we want to generate the names for temp columns in general, we might have to do it separately.